### ISSUE TITLE
xigtxml namespaces get pushed to elements

### ISSUE THREAD

**Issue by goodmami** [goodmami/xigt#25]
If we have a namespace declared, as in the following metadata:

```xml
<metadata xmlns:olac="http://www.language-archives.org/OLAC/1.1/"
          xmlns:dc="http://purl.org/dc/elements/1.1/"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <meta id="md1">
    <dc:subject xsi:type="olac:language" olac:code="ko"/>
    <dc:language xsi:type="olac:language" olac:code="en"/>
  </meta>
</metadata>
```

The namespaces get pushed from their declarations to the elements on reading. This changes the names of the attributes, and is more noticeable when serializing with xigtxml:

```xml
<metadata>
  <meta id="md1">
      <dc:subject xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:ns2="http://www.language-archives.org/OLAC/1.1/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" ns2:code="ko" xsi:type="olac:language" />
      <dc:language xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:ns2="http://www.language-archives.org/OLAC/1.1/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" ns2:code="en" xsi:type="olac:language" />
  </meta>
</metadata>
```

This is actually a documented feature of ElementTree: https://docs.python.org/3/library/xml.etree.elementtree.html#parsing-xml-with-namespaces

It would be better if these could be handled appropriately without expanding the prefixes, but it means the namespaces will need to be stored in the Xigt-internal structures somewhere, making it somewhat more XML-specific.

**Comment by goodmami** [comment]
One solution is to move to lxml, which can track namespaces well. Alternatively, the "start-ns" and "end-ns" events of ElementTree.iterparse() can be captured to manually store the namespace mapping on the elements. The problem with lxml is that it requires bytes-objects instead of strings, so the API may have to change (either pass the filename instead of file handle to xigtxml.load(), or send a file handle opened in "rb" mode).

**Comment by goodmami** [comment]
Resolved by ecb1894b0e8900131169822cceb568afd0882c78
